docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml

```diff
@@ -594,28 +601,136 @@ depending on where they wind up.-->
         package map. To examine a package map, select a package map from the
         list.</para>
 
-        <para>You can select the <emphasis role="bold">Validate Package
-        Map</emphasis> tab to validate a package map.</para>
+        <sect3>
+          <title id="Packagemap_Actions">Packagemap Actions</title>
```

Comment:
Don't think you are looking at the latest commit. This was fixed in a later commit.